components/rt/src/rt_estimator.m

```diff
@@ -1,35 +1,35 @@
 %% Atte Aalto
-
-% change to the root of the file
-pathToFile = fileparts(mfilename('fullpath'));
-if ~isempty(pathToFile)
-    rootFolder = [pathToFile filesep '..'];
-else
-    rootFolder = pwd();
-end
-
-cd(rootFolder);
-addpath(genpath(rootFolder));
-
-% define output folder
-outFolder = [rootFolder filesep 'output'];
-checkFolder(outFolder);
-inFolder = [rootFolder filesep 'input'];
-checkFolder(inFolder);
+% 
```

Comment:
why remove this? these are important in my opinion

Reply:
This gets the input file after the check, when it's already renamed. Do you think the procedure might fail without us noticing, and a second check on Matlab should be kept? If so, fine by me

Reply:
they are indeed very important commands, as MATLAB needs to have the full source path included

Reply:
all right